langchain/src/agents/toolkits/sql/sql.ts

```diff
@@ -44,7 +45,7 @@ export function createSqlAgent(
   args?: SqlCreatePromptArgs
 ) {
   const {
-    prefix = SQL_PREFIX,
+    prefix = toolkit.db.appDataSourceOptions?.type === 'sap' ? SQL_PREFIX_SAPHANA : SQL_PREFIX,
```

Comment:
Let's do this in a separate PR and fix it for all the different SQL dialects?

Reply:
ok - will remove this